docs/quick-start/README.md

```diff
@@ -12,20 +12,20 @@ Or, even faster, you can run:
 npm install @helpscout/hsds-react --save
 ```
 
-## React (v15)
+## React (v16.8+)
 
-Blue has a couple of `peerDependencies`:
+HSDS has a couple of `peerDependencies`:
 
 - [`react`](https://www.npmjs.com/package/react)
 - [`react-dom`](https://www.npmjs.com/package/react-dom)
 
 Make sure you have these installed in your project.
 
-Blue currently runs on **React version 15**. Work is being done to add compatibility with 16+. 💪
+Blue currently runs on **React version 16**.
```

Comment:
`HSDS` instead of `Blue`?

Reply:
Nice catch :), all the guides and dcos will be reviewed and updated soon, I'll leave these changes for now